docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+
+    if (recurrence == null) {
+        console.log("Non-recurring meeting");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the results of the `getAsync` call on a series.
+
+> [!NOTE]
+> In this example, the `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```js
+Recurrence = {
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get recurrence as an attendee
+
+In the following example, an appointment attendee can get the recurrence object of an appointment series given the series, an instance of that series, or a meeting request.
+
+```js
+outputRecurrence(Office.context.mailbox.item);
+
+function outputRecurrence(item) {
+    var recurrence = item.recurrence;
+    var seriesId = item.seriesId;
+
+    if (recurrence == null) {
+        console.log("Non-recurring item");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the `item.recurrence` property.
+
+> [!NOTE]
+> In this example, the `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```js
+Recurrence = {
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Access the recurrence details
+
+Now that you have the recurrence object (either from the callback or `item.recurrence`), you can get the specific properties you want to work with. For example, to get the series start and end dates and times, you use the methods on the `recurrence.seriesTime` property.
+
+```js
+// Get series date and time info
+var seriesTime = recurrence.seriesTime;
+var startTime = recurrence.seriesTime.getStartTime();
+var endTime = recurrence.seriesTime.getEndTime();
+var startDate = recurrence.seriesTime.getStartDate();
+var endDate = recurrence.seriesTime.getEndDate();
+var duration = recurrence.seriesTime.getDuration();
+
+// Get series time zone
+var timeZone = recurrence.recurrenceTimeZone;
+
+// Get recurrence properties
+var recurrenceProperties = recurrence.recurrenceProperties;
+
+// Get recurrence type
+var recurrenceType = recurrence.recurrenceType;
+```
+
+## See also
+
+[`RecurrenceChanged` event](/javascript/api/office/office.eventtype)
```

Comment:
I was wondering, do you want to link to any other items referenced in the article, like SeriesTime, and others?

Reply:
@kbrandl: regarding the link, it's fine. It only gives a 404 because I don't have a similarly named branch in -reference.
@davidchesnut: I wasn't sure about that. It seemed like "See also" would contain references/links to additional topics/resources while SeriesTime etc were already included in the article and so are not additional.